docs/querying-data.rst

```diff
@@ -0,0 +1,207 @@
+
+Kingfisher Process automatically transforms OCDS releases to create compiled release collections,  containing a single compiled release per contracting process populated with the latest value of each field.
+
+Tables in Kingfisher Views can contain summaries of:
+
+* individual releases
+* compiled releases generated by Kingfisher Process
+* compiled releases published as part of an OCDS record
+
+Since most analysis is much easier to perform on compiled releases, we recommend working with compiled releases to begin with.
+
+List all schemas in the database
+--------------------------------
+
+Kingfisher Views creates database schemes to store summary data.
+
+The following query returns a list of schemas in the database:
+
+.. code-block:: sql
+
+  select
+    schema_name
+  from
+    information_schema.schemata;
+
+Kingfisher Views schemas are named ``view_data_*``, if you don't see any such schemas refer to :doc:`cli/index` and use the ``add-view`` command to generate a schema.
+
+Set the schema to query
+-----------------------
+
+The following command sets the current schema to ``view_data_collection_1070_1071_1072`` by making it the first item in the search path:
+
+.. code-block:: sql
+
+  set
+    search_path
+  to
+    view_data_collection_1070_1071_1072, public;
+
+.. note::
+
+  Depending on the tool used to query the database, you may need to include the above command at the start of each query you run.
+
+List the collections in the current schema
+------------------------------------------
+
+The following query lists the collections in the current schema, with the name of the data source and the type of data summarised:
+
+.. code-block:: sql
+
+  select distinct
+      collection_id,
+      source_id,
+      release_type
+  from
+      release_summary
+  join
+      collection
+  on
+      release_summary.collection_id = collection.id
+  order by
+      collection_id desc;
+
+The ``release_type`` column indicates the type of data stored in the collection:
+
+* ``release`` identifies individual releases
+* ``compiled_release`` identifies compiled releases generated by Kingfisher Process
+* ``record`` identifies compiled releases published as part of an OCDS record
+
+Get a top-level summary of contracting processes
+------------------------------------------------
+
+The ``release_summary`` table contains data summarised at the top level of an OCDS release.
+
+Use one of the ``collection_id`` returned by the previous query to filter your results to a single collection. To get a summary of a whole contracting process rather choose a ``compiled_release`` or ``record`` collection.
+
+The following query returns a top-level summary of the first 3 contracting processes in collection ``1060``.
+
+.. code-block:: sql
+
+  select
+      *
+  from
+      release_summary
+  where
+      collection_id = 1060
+  limit 3;
+
+To learn more about the summaries and aggregates in the ``release_summary`` table, refer to the :doc:`database`.
```

Comment:
Please add a reference before the `release_summary` heading (like `.. _release_summary:`). Then, link to it directly as ```:ref:`release_summary` ```. You can do this for all instances of ```:doc:`database` ```

Reply:
Thanks, I was wondering how to do that! Done in https://github.com/open-contracting/kingfisher-views/pull/109/commits/e9666e4517a8ef2b8f56e53da6b6bef795c67a8b